kep/115-Subgroup-support/README.md

```diff
@@ -0,0 +1,202 @@
+# KEP #115 Subgroup Support
+<!--
+This is the title of your KEP. Keep it short, simple, and descriptive. A good
+title can help communicate what the KEP is and should be considered as part of
+any review.
+-->
+
+<!--
+A table of contents is helpful for quickly jumping to sections of a KEP and for
+highlighting any additional information provided beyond the standard KEP
+template.
+
+Ensure the TOC is wrapped with
+  <code>&lt;!-- toc --&rt;&lt;!-- /toc --&rt;</code>
+tags, and then generate with `hack/update-toc.sh`.
+-->
+
+<!-- toc -->
+- [KEP #115 Subgroup Support](#kep-115-subgroup-support)
+  - [Motivation](#motivation)
+  - [Proposal](#proposal)
+    - [User Stories (Optional)](#user-stories-optional)
+      - [Story 1](#story-1)
+    - [Risks and Mitigations](#risks-and-mitigations)
+  - [Design Details](#design-details)
+    - [LeaderWorkerSet API](#leaderworkerset-api)
+    - [Exclusive Topology Support](#exclusive-topology-support)
+    - [Implementation](#implementation)
+    - [TPU Environment Variable Injection](#tpu-environment-variable-injection)
+    - [Test Plan](#test-plan)
+      - [Unit Tests](#unit-tests)
+      - [Integration tests](#integration-tests)
+  - [Alternatives](#alternatives)
+    - [Only Set Pod-Affinity on workers with `subgroup-worker-index=0`](#only-set-pod-affinity-on-workers-with-subgroup-worker-index0)
+<!-- /toc -->
+
+## Motivation
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+Disaggregated serving is an optimization made for LLM inference workloads. It takes advantage of the fact that the two phases of inference have different characteristics and thus it can be beneficial to run them on different machines. State of the art LLM serving frameworks such as [vLLM](https://github.com/vllm-project/vllm/issues/2472) are already adding support for this optimization based on the paper released by [Microsoft](https://www.microsoft.com/en-us/research/publication/splitwise-efficient-generative-llm-inference-using-phase-splitting/). 
+
+LeaderWorkerSet does not currently support having a group for each phase.
```

Comment:
```suggestion
LeaderWorkerSet does not currently support having a group for each phase.
```
This KEP is to have LeaderWorkerSet to support running both prefill/decode servers in one pod group (aka one replica)

Reply:
Fixed